app/views/admin/users/edit.html.erb

```diff
@@ -46,6 +46,7 @@
 </div>
 
 <%= render "notes" %>
+<%= render "reactions" %>
```

Comment:
Maybe call the file `negative_reactions`?

Reply:
I'll make this change now!